gensim/models/word2vec.py

```diff
@@ -606,7 +606,7 @@ def load_word2vec_format(cls, fname, fvocab=None, binary=False, norm_only=True):
                     result.syn0[line_no] = fromstring(fin.read(binary_len), dtype=REAL)
             else:
                 for line_no, line in enumerate(fin):
-                    parts = utils.to_unicode(line).split()
+                    parts = utils.to_unicode(line.rstrip()).split(" ")
```

Comment:
The space problem is within the word, which is the first token. You can for example have something like 

hello world 0.234 0.345 ....

and assume that the space between hello and world is not in [ \t\n] 

rstrip is inconsequential. I needed it because I only used " " to split. If you use more than whitespace it may not be needed.